pom.xml

```diff
@@ -87,9 +87,9 @@
 		</dependency>
 
 		<dependency>
-			<groupId>javax.jmdns</groupId>
+			<groupId>org.jmdns</groupId>
 			<artifactId>jmdns</artifactId>
-			<version>3.4.1</version>
```

Comment:
hmm... breaking change to bump a minor version of a dependency? May need to wait for 1.2.0 release?

Reply:
Why is changing the maven organization from the abandoned name to the maintained name a breaking change (the implemented API did not change in a breaking way)?